doc/source/howtos/encrypt-traffic.rst

```diff
@@ -0,0 +1,307 @@
+   ssl-key=client-key.pem
+
+For more information, see the relevant sections about SSL certificates
+in `Galera Cluster Documentation
+<http://galeracluster.com/documentation-webpages/sslcert.html>`_.
+and `MySQL Server Documentation
+<http://dev.mysql.com/doc/refman/5.7/en/creating-ssl-files-using-openssl.html>`_.
+
+SST Traffic
+===========
+
+This refers to full data transfer
+that usually occurs when a new node (JOINER) joins the cluster
+and receives data from an existing node (DONOR).
+
+For more information, see :ref:`state_snapshot_transfer`.
+
+When copying encrypted data via SST,
+the keyring must be sent over with the files for decryption.
+Make sure that the following options are set in :file:`my.cnf` on all nodes:
+
+.. code-block:: text
+
+   early-plugin-load=keyring_file.so
+   keyring-file-data=/path/to/keyring/file
+
+.. warning:: The cluster will not work if keyring configuration across nodes
+   is different.
+
+The following SST methods are available:
+``rsync``, ``mysqldump``, and ``xtrabackup``.
+
+rsync
+-----
+
+This SST method does not support encryption.
+Avoid using this method if you need to secure traffic
+between DONOR and JOINER nodes.
+
+mysqldump
+---------
+
+This SST method dumps data from DONOR and imports it to JOINER.
+Encryption in this case is performed using the SSL certificates configured
+for :ref:`secure MySQL client-server communication <client-server-encryption>`.
+
+Here is how to perform secure SST using ``mysqldump``:
+
+1. Ensure that the DONOR node is configured for SSL encryption
+   (both ``[mysqld]`` and ``[client]`` sections).
+   For more information, see :ref:`client-server-encryption`.
+
+#. Create an SSL user on the DONOR node.
+
+   .. code-block:: sql
+
+      mysql> GRANT USAGE ON *.* TO 'sslsst' REQUIRE SSL;
+
+#. Specify the DONOR superuser credentials
+   in the :variable:`wsrep_sst_auth` variable.
+
+#. Start the JOINER node without Galera library (``--wsrep_provider=none``)
+   and create an SSL user with the same name and grants as on the DONOR node.
+
+#. Configure SSL encryption on JOINER node with the same parameters as DONOR
+   (both ``[mysqld]`` and ``[client]`` sections).
+
+#. Restart JOINER node with Galera library.
+
+If you do everything correctly,
+``mysqldump`` will connect to DONOR using SSL user,
+generate a dump file, and import it to JOINER node.
+
+For more information, see `the relevant section in Galera Cluster documentation <http://galeracluster.com/documentation-webpages/mysqldump.html>`_.
+
+xtrabackup
+----------
+
+This is the default SST method,
+which uses Percona XtraBackup to perform non-blocking transfer of files.
+For more information,
+see :ref:`xtrabackup_sst`.
+
+Encryption mode for this method is selected using the :option:`encrypt` option.
+Depending on the mode you select, other options will be required.
+
+* To enable built-in XtraBackup encryption, use the following configuration:
+
+  .. code-block:: text
+
+     [sst]
```

Comment:
Please add Bullet.

Option/Alternative-1
Option/Alternative-2
Option/Alternative-3
Option/Alternative-4

This makes it more clear that either of it needs to be selected.